tools/docker-compose/ansible/roles/sources/templates/receptor-awx.conf.j2

```diff
@@ -7,6 +7,13 @@
 - tcp-listener:
     port: 2222
 
+- work-signing:
+    privatekey: /etc/receptor/work_private_key.pem
+    tokenexpiration: 1m
```

Comment:
when awx_1 makes a connection to receptor_1, it issues a payload with the "work submit" command. This payload has a json web token that expires after 1 minute. so, it has 1 whole minute to send that "work submit" request to receptor_1.

It only generates the token at the very last second, after it has already made a valid connection to receptor_1. Therefore, even if receptor_1 is down temporarily, it should only start counting when it comes back online.

Reply:
Okay, so this payload is just telling the execution node "run `ansible-runner worker`", right? _After_ that, the controller node and the execution node will be chatting it up over the socket. This will not require that the streaming of the job directory from the controller node to the execution node has finished in 1 minute, right? Because it will almost certainly take longer in cases.